enforcer/src/keystate/keystate_ds_submit_task.c

```diff
@@ -48,7 +48,7 @@ keystate_ds_submit_task_perform(task_type *task)
 		KEY_DATA_DS_AT_PARENT_SUBMIT, KEY_DATA_DS_AT_PARENT_SUBMITTED,
 		(engine_type*)task->context);
 	task_cleanup(task);
-	flush_enforce_task((engine_type*)task->context, 0);
+	flush_enforce_task((engine_type*)task->context, time_now());
```

Comment:
Okay, I was in a rush. I'll revert it before the test completes.  It does not have any influence on the other tests (nor this test), so the remainder can be reviewed.